pkg/network/usm/utils/debugger.go

```diff
@@ -266,7 +280,14 @@ func (d *tlsDebugger) runAttacherCallback(w http.ResponseWriter, r *http.Request
 	}
 
 	d.mux.Lock()
-	attacher, ok := d.attachers[reqBody.Type]
+	moduleAttachers, ok := d.attachers[moduleName]
+	if !ok {
+		d.mux.Unlock()
+		w.WriteHeader(http.StatusBadRequest)
+		fmt.Fprintf(w, "module %q is unrecognized", moduleName)
```

Comment:
Maybe we can add unit tests to validate this flow, just to ensure that when we use a module that doesn't exist, everything behaves as expected.

Reply:
done